hardware-handler/nightmare_gazebo/src/joint_state_redirector.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import time
+import numpy as np
+
+import rospy
+from std_msgs.msg import Header
+from sensor_msgs.msg import JointState
+from std_msgs.msg import Float64
+
+rostopic = ['/nightmare/leg1coxa_position_controller/command',
+            '/nightmare/leg1femur_position_controller/command',
+            '/nightmare/leg1tibia_position_controller/command',
+            '/nightmare/leg2coxa_position_controller/command',
+            '/nightmare/leg2femur_position_controller/command',
+            '/nightmare/leg2tibia_position_controller/command',
+            '/nightmare/leg3coxa_position_controller/command',
+            '/nightmare/leg3femur_position_controller/command',
+            '/nightmare/leg3tibia_position_controller/command',
+            '/nightmare/leg4coxa_position_controller/command',
+            '/nightmare/leg4femur_position_controller/command',
+            '/nightmare/leg4tibia_position_controller/command',
+            '/nightmare/leg5coxa_position_controller/command',
+            '/nightmare/leg5femur_position_controller/command',
+            '/nightmare/leg5tibia_position_controller/command',
+            '/nightmare/leg6coxa_position_controller/command',
+            '/nightmare/leg6femur_position_controller/command',
+            '/nightmare/leg6tibia_position_controller/command',
+            '/nightmare/tail_joint_position_controller/command']
+
+JOINTSTATE_MSG = JointState(header=Header(),
+                            name=['leg1coxa', 'leg1femur', 'leg1tibia',
+                                  'leg2coxa', 'leg2femur', 'leg2tibia',
+                                  'leg3coxa', 'leg3femur', 'leg3tibia',
+                                  'leg4coxa', 'leg4femur', 'leg4tibia',
+                                  'leg5coxa', 'leg5femur', 'leg5tibia',
+                                  'leg6coxa', 'leg6femur', 'leg6tibia', 'tail_joint'],
+                            velocity=[],
+                            effort=[])
+
+
+class Node():
+    def __init__(self):
```

Comment:
same move values of instance variables to constructor parameters

Reply:
no variables need to be initialized via parameters here, it should be a singleton but can't be bothered